webpack/config.dev.js

```diff
@@ -0,0 +1,14 @@
+var webpack = require('webpack');
+var merge = require('lodash.merge');
+var baseConfig = require('./config.base');
+
+module.exports = merge(baseConfig, {
+  devtool: 'cheap-module-eval-source-map',
+  entry: [
+    'webpack-hot-middleware/client'
+  ].concat(baseConfig.entry),
```

Comment:
I might be misunderstanding, but as long as we use `concat` for array extension, isn't that behavior a non-issue here?